platform/linux-generic/include/odp_internal.h

```diff
@@ -135,6 +135,33 @@ uint64_t odp_cpufreq_id(const char *filename, int id);
 uint64_t odp_cpu_hz_current(int id);
 void sys_info_print_arch(void);
 
+#define ARRAY_SIZE(x) (sizeof(x) / sizeof((x)[0]))
+
+#define MIN(a, b)				\
+	({					\
+		__typeof__(a) tmp_a = (a);	\
+		__typeof__(b) tmp_b = (b);	\
+		tmp_a < tmp_b ? tmp_a : tmp_b;	\
+	})
+#define MAX(a, b)				\
```

Comment:
Insert blank link before this macro for consistency with the rest of the macros defined here.

Reply:
Done.